src/fidesops/service/authentication/authentication_strategy.py

```diff
@@ -11,7 +11,7 @@ class AuthenticationStrategy(ABC):
 
     @abstractmethod
     def add_authentication(
-        self, request: PreparedRequest, secrets: Dict[str, Any]
+        self, request: PreparedRequest, connection_config: ConnectionConfig
```

Comment:
Changes the signature for `add_authentication` to use `connection_config` instead of `secrets`. It is more flexible to pass in the entire connection config than just a subset of it.